.github/workflows/releaseTrigger.yml

```diff
@@ -2,18 +2,14 @@
 name: Release
 
 on:
-  create:
-    tags:
-      - 'releaseTrigger'
+  workflow_dispatch:
 
 jobs:
   deploy:
     runs-on: ubuntu-latest
     name: releasing
     steps:
       - uses: actions/checkout@v2
-        with:
-          ref: master
       - name: Wait on builds
```

Comment:
@aepfli sorry if I was imprecise, what I actually wanted to ask: shall we remove this step since there is no other build to wait for? What's your say on that?

Reply:
i would leave it actually, because theoretically, somebody could hit release on master branch, although all the checks are not finished.